cloud

```diff
@@ -13,8 +13,9 @@
 set -eEuCo pipefail
 shopt -s extglob
 
+declare -r TMPDIR_ORIG=${TMPDIR:-}
 # The directory where all temporary files should be created
-export TMPDIR=$(mktemp -d -t "kcidb_cloud.XXXXXXXXXX")
+declare -r TMPDIR=$(mktemp -d -t "kcidb_cloud.XXXXXXXXXX")
```

Comment:
Please do not change this `export` to `declare -r`. We *need* this exported.

Reply:
Okay